great_expectations/core/expectation_suite.py

```diff
@@ -518,42 +518,21 @@ def patch_expectation(
         self.expectations[found_expectation_indexes[0]].patch(op, path, value)
         return self.expectations[found_expectation_indexes[0]]
 
-    def add_expectation(
+    def _add_expectation(
```

Comment:
please add a comment to this method, saying that it is a private method for adding expectations that allows for usage events to be suppressed during internal processing (for rendering and validation)

Reply:
done 🙇🏼 sorry for missing this one